src/app/tabs/compileTab/compilerContainer.js

```diff
@@ -389,6 +409,14 @@ class CompilerContainer {
     }
   }
 
+  _retrieveOS () {
+    let osName = 'Unknown OS'
+    if (navigator.platform.indexOf('Win') !== -1) osName = 'Windows'
+    if (navigator.platform.indexOf('Mac') !== -1) osName = 'MacOS'
```

Comment:
Not sure if we should check for MacOS as all versions are working fine with it. Also I would prefer to use `if - else if - else`

Reply:
updated